docs/static/releasenotes.asciidoc

```diff
@@ -3,9 +3,52 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-5-4-2,Logstash 5.4.2>>
 * <<logstash-5-4-1,Logstash 5.4.1>>
 * <<logstash-5-4-0,Logstash 5.4.0>>
 
+[[logstash-5-4-2]]
+=== Logstash 5.4.2 Release Notes
+
+* We now handle connection refused errors when installing a plugin ({lsissue}6529[Issue 6529]).
+* Fixed an issue where environment variables specified in Elasticsearch Output host configuration 
+  wasn't resolved correctly ({lsissue}6696[Issue 6696]).
+* Fixed a concurrency issue in the persistent queues feature ({lsissue}7382[Issue 7382]).
+
+[float]
+==== Input Plugins
+
+*`HTTP Poller`*:
+
+* Added a top level user/password config option that apply to all URLs by default.
```

Comment:
Change to: "Added a top-level user/password config option that applies to all URLs by default."